src/sorting/main.c

```diff
@@ -0,0 +1,26 @@
+#include"sorting/sorting.h"
+
+int main() {
+  int arr[SIZE], temp_arr[SIZE], i = 0;
```

Comment:
**Use pointer and malloc instead of fixed-size arrays.** 

The program should run:
./\<exe-name\> \<space-separated-numbers\>
Inside `main` function copy these numbers into array created using malloc.